code/modules/mob/living/carbon/xenomorph/castes/widow/abilities_widow.dm

```diff
@@ -0,0 +1,352 @@
+	plasma_cost = 500
+	cooldown_timer = 60 SECONDS
+	keybind_signal = COMSIG_XENOABILITY_SPIDER_SWARM
+	/// The spiderling we are controlling right now
+	var/mob/living/carbon/xenomorph/spiderling/current_controlling_spiderling
+	/// how many spiderlings should spawn to replace widow
+	var/amount_of_spiderlings = 2
+
+/datum/action/xeno_action/spider_swarm/action_activate()
+	. = ..()
+	if(!do_after(owner, 3 SECONDS, TRUE, owner, BUSY_ICON_DANGER))
+		return fail_activate()
+	/// We store the spiderling that we want to posses in order to make the other spiderlings follow it
+	current_controlling_spiderling = new(get_turf(owner), owner)
+	SEND_SIGNAL(owner, COMSIG_ESCORTED_ATOM_CHANGING, current_controlling_spiderling)
+	var/datum/action/xeno_action/return_to_mother/new_action = new
+	new_action.mother = owner
+	new_action.give_action(current_controlling_spiderling)
+	owner.mind.transfer_to(current_controlling_spiderling)
+	ADD_TRAIT(owner, TRAIT_POSSESSING, WIDOW_ABILITY_TRAIT)
+	new /obj/structure/xeno/widow_pod(get_turf(owner), owner)
+	/// We want to access the spiderlings list and therefore have this
+	var/datum/action/xeno_action/create_spiderling/create_spiderling_action = owner.actions_by_path[/datum/action/xeno_action/create_spiderling]
+
+	for(var/spawned_spiderlings = 0 to amount_of_spiderlings)
+		var/mob/living/carbon/xenomorph/spiderling/new_spiderling = new (current_controlling_spiderling.loc, current_controlling_spiderling)
+		/// here we add the created spiderligns to the list
+		create_spiderling_action.add_spiderling(new_spiderling)
+
+	create_spiderling_action.add_spiderling(current_controlling_spiderling)
+	succeed_activate()
+	add_cooldown()
+
+/// This happens whenever the spiderling that we control dies and there are spiderlings left to control
+/datum/action/xeno_action/spider_swarm/proc/switch_to_next_spiderling(mob/living/carbon/xenomorph/spiderling/spiderling)
+	SEND_SIGNAL(current_controlling_spiderling, COMSIG_ESCORTED_ATOM_CHANGING, spiderling)
+	var/datum/action/xeno_action/return_to_mother/new_action = new
+	new_action.mother = owner
+	new_action.give_action(spiderling)
+	current_controlling_spiderling.mind.transfer_to(spiderling)
+	current_controlling_spiderling = spiderling
+
+/// Put the player back in widow
+/datum/action/xeno_action/spider_swarm/proc/switch_to_mother()
+	current_controlling_spiderling.mind.transfer_to(owner)
+	var/datum/action/xeno_action/create_spiderling/create_spiderling_action = owner.actions_by_path[/datum/action/xeno_action/create_spiderling]
+	for(var/mob/living/carbon/xenomorph/spiderling/spiderlings_to_kill AS in create_spiderling_action.spiderlings)
+		spiderlings_to_kill.death(FALSE, "", TRUE)
+	REMOVE_TRAIT(owner, TRAIT_POSSESSING, WIDOW_ABILITY_TRAIT)
+
+/// This ability is being given to spiderlings so we can always return to widow
+/datum/action/xeno_action/return_to_mother
+	name = "Return to Widow"
+	ability_name = "Return to Widow"
+	mechanics_text = " Return to Widow"
```

Comment:
```suggestion
	mechanics_text = "Return control to the Widow. This action will kill ALL of your active spiderlings. Resist while in the pod to break free."
```

Reply:
seriously, you need to let the player know that you must press the resist key to get out once you gain control to the widow